### PR TITLE
Feature/plugin support

### PR DESCRIPTION
**Pull request by StephenHodgson**
# XRTK - Mixed Reality Toolkit Pull Request

## Overview

Added plugin utility

### CODE REVIEW

File: XRTK.Ultraleap/Packages/com.xrtk.ultraleap/Editor/UltraleapPluginUtility.cs [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        }
+
+        [MenuItem("Mixed Reality Toolkit/Tools/Ultraleap/Reimport Plugins", true)]
+        private static bool UpdatePluginValidation() => Directory.Exists(NativeRootPath);
+
+        [MenuItem("Mixed Reality Toolkit/Tools/Ultraleap/Reimport Plugins", false)]
+        private static void UpdatePlugins()
+        {
+            if (EditorUtility.DisplayDialog("Attention!",
+                "In order to reimport the SteamVR plugins, we'll need to restart the editor, is this ok?", "Restart", "Cancel"))
```
Comment (StephenHodgson) [inline XRTK.Ultraleap/Packages/com.xrtk.ultraleap/Editor/UltraleapPluginUtility.cs]:
```suggestion
                "In order to reimport the Ultraleap plugins, we'll need to restart the editor, is this ok?", "Restart", "Cancel"))
```

**Review by FejZa — DISMISSED** [review]

**Comment by StephenHodgson** [comment]
Gotta get it passing CI/CD first. I think I need to open a PR on the main repo to add an assembly definition to the LeapCSharp files

**Comment by StephenHodgson** [comment]
@acolgan I'd also like your blessing about this strategy as well.

**Comment by StephenHodgson** [comment]
Blocked by https://github.com/leapmotion/UnityModules/pull/1111

**Comment by vabrador** [comment]
leapmotion/UnityModules#1111 has been merged.

**Review by FejZa — APPROVED** [review]

**Pull request merged**